### ISSUE TITLE
[Bug?] Sequence Diagram Actor Font Family Is Serif

### ISSUE THREAD

**Issue by KSR-Yasuda** [mermaid-js/mermaid#2689]
[Bug?] Sequence Diagram Actor Font Family is Serif

**Describe the bug**

In sequence diagram, PNG output gets actor font family some of *serif* font.

Isn't it [*sans-serif* font by default](https://github.com/mermaid-js/mermaid/blob/develop/docs/sequenceDiagram.md#:~:text=14-,actorFontFamily,-Sets%20the%20font)?

I checked on Windows and Ubuntu, so it would not be an OS specific problem.

**To Reproduce**

Convert `sequenceDiagram` into a PNG.

```text
%%{ init : { "theme" : "default" }}%%

sequenceDiagram

%%==============================================================================

participant A as Alice
participant B as Bob
actor       C as Carole
actor       T as Tuesday

%%==============================================================================

A ->> B : Message
```

```bash
% mmdc -V
8.13.10

% mmdc -i font.mmd -o font.png
```

![font.png](https://user-images.githubusercontent.com/31273423/152721003-bbf71980-bf33-4144-a95d-6680d615d258.png)

**Expected behavior**

Use *sans-serif* font as actor font family.

**Desktop (please complete the following information):**

* OS      : Windows 10 Pro 21H1 / Ubuntu 18.04.6 LTS
* Version : [mmdc][] v8.13.10

[mmdc]: https://www.npmjs.com/package/@mermaid-js/mermaid-cli

**Comment by dbartholomae** [comment]
Trying my hand at this one :)

**Comment by dbartholomae** [comment]
The problem isn't specific to png and happens also e. g. for svg.

Looks like the default font family for actors should be `Open Sans; sans-serif`, not `Open-sans, "sans-serif"`, as `Open-sans` does not exist and the fallback doesn't work with quotation marks.

I'll open a PR shortly to address this.

**Comment by KSR-Yasuda** [comment]
Is this working for png output?

I still get serif font for actor name on mmdc v8.14.0.
I think it contains the patch #2729...

**Comment by KSR-Yasuda** [comment]
Confirmed the fix on mmdc v9.0.0.
